index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital@1&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Mo&lsquo;athal&lsquo;s Portfolio</title>
+</head>
+
+<body>
+    <div class="container flex">
+        <header class="toolbar">
+            <span class="theme logo"> Mo&lsquo;athal S. Kachi</span>
+            <button></button>
```

Comment:
- [ ] Please wrap this hamburger button in a `nav` tag. The hamburger button contains navigation links to different places on your portfolio as you will see in the future 👌